worksheet_A/TerminalHacking.cpp

```diff
@@ -3,9 +3,13 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <stdio.h>
+#include <set>
```

Comment:
Better to add these to `stdafx.h` to take advantage of precompiled headers (i.e. faster compile times).